ansible/roles/products/hand-e/server/desktop-icons/tasks/hand-and-arm.yml

```diff
@@ -39,24 +47,24 @@
+    launch_file_content_arm_hand_left_sim: "{{ launch_file_input_arm_hand_left }} external_control_loop:=false sim:=true arm_speed_scale:=0.7 initial_y:={{ initial_y_left }} initial_z:={{ initial_z }}"
+    launch_file_content_arm_hand_bimanual_sim: "{{ launch_file_input_arm_hand_bimanual }} external_control_loop:=false sim:=true arm_speed_scale:=0.7 arm_1_z:={{ initial_z }} arm_2_z:={{ initial_z }} arm_y_separation:={{ arm_y_separation }}"
 
 - name: Set arm+hand GUI launch file content
   set_fact:
-    launch_file_content_arm_hand_right: "{{ launch_file_input_arm_hand_right }} external_control_loop:=true sim:=false arm_speed_scale:=0.7 scene:=true"
-    launch_file_content_arm_hand_left: "{{ launch_file_input_arm_hand_left }} external_control_loop:=true sim:=false arm_speed_scale:=0.7 scene:=true"
+    launch_file_content_arm_hand_right: "{{ launch_file_input_arm_hand_right }} external_control_loop:=true sim:=false arm_speed_scale:=0.7"
+    launch_file_content_arm_hand_left: "{{ launch_file_input_arm_hand_left }} external_control_loop:=true sim:=false arm_speed_scale:=0.7"
     launch_file_content_arm_hand_bimanual: "{{ launch_file_input_arm_hand_bimanual }} external_control_loop:=true sim:=false arm_speed_scale:=0.7"
 
 - name: Set arm+hand GUI launch file content for custom scene
   set_fact:
-    launch_file_content_arm_hand_right: "{{ launch_file_content_arm_hand_right }} scene_file:='{{ scene_file }}'"
-    launch_file_content_arm_hand_left: "{{ launch_file_content_arm_hand_left }} scene_file:='{{ scene_file }}'"
-    launch_file_content_arm_hand_bimanual: "{{ launch_file_content_arm_hand_bimanual }} scene_file:='{{ scene_file }}'"
-    launch_file_content_arm_hand_right_sim: "{{ launch_file_content_arm_hand_right_sim }} scene_file:='{{ scene_file }}'"
-    launch_file_content_arm_hand_left_sim: "{{ launch_file_content_arm_hand_left_sim }} scene_file:='{{ scene_file }}'"
-    launch_file_content_arm_hand_bimanual_sim: "{{ launch_file_content_arm_hand_bimanual_sim }} scene_file:='{{ scene_file }}'"
+    launch_file_content_arm_hand_right: "{{ launch_file_content_arm_hand_right }} scene:=true scene_file:='{{ scene_file }}' world:='{{ scene_file | replace('.scene','.world') | replace('/scenes/','/worlds/') }}'"
+    launch_file_content_arm_hand_left: "{{ launch_file_content_arm_hand_left }} scene:=true scene_file:='{{ scene_file }}' world:='{{ scene_file | replace('.scene','.world') | replace('/scenes/','/worlds/') }}'"
+    launch_file_content_arm_hand_bimanual: "{{ launch_file_content_arm_hand_bimanual }} scene:=true scene_file:='{{ scene_file }}' world:='{{ scene_file | replace('.scene','.world') | replace('/scenes/','/worlds/') }}'"
+    launch_file_content_arm_hand_right_sim: "{{ launch_file_content_arm_hand_right_sim }} scene:=true scene_file:='{{ scene_file }}' world:='{{ scene_file | replace('.scene','.world') | replace('/scenes/','/worlds/') }}'"
```

Comment:
instead of this only add the name of the file `scene_file_name=customer_stand_bimanual` 

e.g.
`launch_file_content_arm_hand_right_sim: "{{ launch_file_content_arm_hand_right_sim }} scene:=true scene_file:='$(find sr_description_common)/scenes/{{ scene_file_name }}.scene' world:='"$(find sr_description_common)/worlds/{{ scene_file_name }}.world"'"`